nautobot/extras/jobs.py

```diff
@@ -1225,36 +1189,24 @@ def _run_job():
         # Perform any post-run tasks
         # 2.0 TODO Remove post_run() method entirely
         job.active_test = "post_run"
-        try:
-            output = job.post_run()
-        except Exception as exc:
-            stacktrace = traceback.format_exc()
-            message = (
-                f"An exception occurred during job post_run(): `{type(exc).__name__}: {exc}`\n```\n{stacktrace}\n```"
-            )
-            output = message
-            job.log_failure(message=message)
-            job_result.set_status(JobResultStatusChoices.STATUS_ERRORED)
-        finally:
-            if output:
-                job.results["output"] += "\n" + str(output)
+        output = job.post_run()
```

Comment:
The try/except was added specifically for #1807 - are exceptions captured/logged differently now?

Reply:
Yes. We should allow all exceptions to bubble up and they are handled by Celery and then the traceback gets captured.